examples/benchmark/run.rs

```diff
@@ -25,7 +13,7 @@ async fn main() -> eyre::Result<()> {
     let dataflow = Path::new("dataflow.yml");
     build_dataflow(dataflow).await?;
 
-    dora_daemon::Daemon::run_dataflow(dataflow).await?;
+    run_dataflow(dataflow).await?;
```

Comment:
So basically, the issue with running the daemon directly is that we use `std::env::current_exe()` to spawn runtime node. But in case of examples the current exe is the `run` binary and not the cli which create an infinite loop. I think that you fixed that with adding a runtime args in the example. But I feel like this is a bit hacky, and was very confusing when I was refactoring the cli.